src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article article />
```

Comment:
```suggestion
  <Article {...article} />
```
Сейчас у тебя не передаются нужные параметры в компонент